client/actions/dashboard-action-creators.js

```diff
@@ -10,22 +10,24 @@ export function dashboardSearchSubmitted(streetObject) {
 
         streetsApi.getStreet(streetObject.placeId)
             .then(
-                response => dispatch(dashboardSearchSucceeded(response)),
-                error => dispatch(dashboardSearchFailed(error)),
+                response => dispatch(dashboardSearchSucceeded(streetObject, response)),
```

Comment:
What I want is, to pass the data (in this case, the selected street) not only if the street exist. So we can still display the name and the location on map to the user.

Lemme know if you have a better solution approach.